repos/system_upgrade/common/libraries/rpms.py

```diff
@@ -22,18 +22,24 @@ class LeappComponents(object):
         LeappComponents.FRAMEWORK: {'7': {'pkgs': ['leapp', 'python2-leapp'],
                                           'deps': ['leapp-deps']},
                                     '8': {'pkgs': ['leapp', 'python3-leapp'],
+                                          'deps': ['leapp-deps']},
+                                    '9': {'pkgs': ['leapp', 'python3-leapp'],
                                           'deps': ['leapp-deps']}
                                     },
         LeappComponents.REPOSITORY: {'7': {'pkgs': ['leapp-upgrade-el7toel8'],
                                            'deps': ['leapp-upgrade-el7toel8-deps']},
                                      '8': {'pkgs': ['leapp-upgrade-el8toel9'],
-                                           'deps': ['leapp-upgrade-el8toel9-deps']}
+                                           'deps': ['leapp-upgrade-el8toel9-deps']},
+                                     '9': {'pkgs': ['leapp-upgrade-el9toel10'],
+                                           'deps': ['leapp-upgrade-el9toel10-deps']}
                                      },
         LeappComponents.COCKPIT: {'7': {'pkgs': ['cockpit-leapp']},
-                                  '8': {'pkgs': ['cockpit-leapp']}
+                                  '8': {'pkgs': ['cockpit-leapp']},
+                                  '9': {'pkgs': ['']}
```

Comment:
Why `['']` and not an empty list?

Reply:
good point, empty list is definitely expected. no idea why I put empty string inside /o\